src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -93,19 +102,15 @@ export const PartyMap: React.FC = () => {
     }
   }, [currentRoom, selectRoom]);
 
-  // @debt We used to specify the explicit venue that we wanted the users for,
-  //   but lost that functionality in #1042, so this may no longer be correct..
-  const usersInVenue = usePartygoers();
-
-  if (!user) return <>Loading..</>;
+  if (!user || !profile) return <>Loading..</>;
 
   return (
     <div className="party-venue-container">
       <Map
         user={user}
-        profileData={profile?.data}
```

Comment:
The new fix for the breaking  changes is being removed here, is this what you want?